bin/geninfo

```diff
@@ -2200,7 +2240,7 @@ sub intermediate_text_to_info($$$)
 				print($fd "FNDA:$2,$3\n");
 			} elsif ($line =~ /^branch:(\d+),(taken|nottaken|notexec)/) {
 				next if (!$br_coverage || $excl->{$1} ||
-					 $brexcl->{$1});
+					 (defined($brexcl->{$1}) && ($brexcl->{$1} == 1)));
```

Comment:
The warnings are better emitted when markers are found, for two reasons: 1) reduce the number of warnings (which could happen almost on every C++ statement) and 2) avoid strange behavior because of overlapping of markers (since "normal" branch exclusion markers have preference).